.gitattributes

```diff
@@ -1,30 +1,3 @@
-# Automatically detect text files and perform LF line ending normalization
-* text=auto
-
-# Files you want to always be normalized and converted
-# to native line endings on checkout
-*.c text
-*.cpp text
-*.h text
-*.in text
-*.dsp text
-*.iss text
-*.rc text
-*.cpf text
-*.sc text
-*.cmd text
-*.sh text
-*.txt text
-
-# Files that will always have CRLF line endings on checkout
-*.sln text eol=crlf
```

Comment:
Referenced in the second commit.

There's no reason to force CRLF for Visual Studio files, which are flexible to LF anyway.

The Git SCM _almost always_ should store as LF.  When you check out to Windows, the Git *nix server converts these to CRLF for you automatically, so there really is nothing to gain from forcing the VC files to CRLF, especially if some program on Linux or whatever had a program that could use these seemingly Microsoft-specific files.  Nothing about them mandates a requirement for forcing CR-LF when Git for Windows is already automated to presuppose this.